### DISCUSSION TITLE
how could I decrease the size of the polygon's line width?

### DISCUSSION THREAD

**Discussion by GustavoSantosBahia** [3b1b/manim#1376] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/68217264/107493307-35e87980-6b6c-11eb-8a4b-5b700b1696ca.png)

how could I decrease the size of the polygon's line width?

**Comment by sahilmakhijani** [comment, marked as answer]
You could either pass the ```stroke_width``` (Default is 4) parameter while creating a polygon object or you could use ```polygon.set_stroke(width={WIDTH})``` just replace {WIDTH} with whatever value you want.
